docs/sources/reference/builder.md

```diff
@@ -626,8 +626,7 @@ ENTRYPOINT ["/usr/sbin/apache2ctl", "-D", "FOREGROUND"]
 
 If you need to write a starter script for a single executable, you can ensure that
 the final executable receives the Unix signals by using `exec` and `gosu`
-(see [the Dockerfile best practices](/articles/dockerfile_best-practices/#entrypoint)
-for more details):
```

Comment:
Oh, okay. I see there's a link to the Best Practices doc at the top of the page, so I'm fine removing this one, then.